src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/AboutListener.java

```diff
@@ -14,27 +17,79 @@
  * @author Glib Sitiugin, Anton Sitkovets, Marco Cirillo
  * @since 2015-03-19
  */
- public class AboutListener implements ActionListener {
+ public class AboutListener extends JPanel implements ActionListener{
```

Comment:
Should a `Listener` also be a `JPanel`? This class sounds like it should be broken up.